webapp/.env

```diff
@@ -0,0 +1 @@
+PORT=1337
```

Comment:
We might need to write a document explaining the different endpoints and PORTS used for the server, webapp, mobile ,etc . But that can be done later.

Reply:
For sure, that should be on docs.